actors/evm/src/interpreter/system.rs

```diff
@@ -385,6 +434,49 @@ impl<'r, RT: Runtime> System<'r, RT> {
         Ok(())
     }
 
+    /// Returns the current transient data lifespan based on the execution environment.
+    pub fn get_current_transient_data_lifespan(rt: &RT) -> Option<TransientDataLifespan> {
```

Comment:
This should just be a free function.

Reply:
I added that here a47c21e36338f087a92762a8bb3cc48acc2b8e99 but it caused some tests to fail which I updated here 4feec0fba90337dc6cffaa24837a801fe5032efc